indico/core/fossils/registration.py

```diff
@@ -812,3 +812,12 @@ def getMiscellaneousGroupList(self):
         Get Miscellaneous group list
         """
     getMiscellaneousGroupList.result = IRegFormMiscellaneousInfoGroupFullFossil
+
+    def getStatuses(self):
+        """
```

Comment:
no need for a docstring here. it's a legacy thing and just defines an interface anyway